core/unit_test/TestWithoutInitializing.hpp

```diff
@@ -239,3 +215,58 @@ TEST(TEST_CATEGORY, deep_copy_zero_memset) {
   ASSERT_TRUE(success);
   listen_tool_events(Config::DisableAll());
 }
+
+// FIXME_OPENMPTARGET
+#ifndef KOKKOS_ENABLE_OPENMPTARGET
+TEST(TEST_CATEGORY, view_allocation_int) {
+  if (!Kokkos::SpaceAccessibility<
+          Kokkos::HostSpace,
+          typename TEST_EXECSPACE::memory_space>::accessible) {
+    using namespace Kokkos::Test::Tools;
+    listen_tool_events(Config::EnableAll());
+    using view_type = Kokkos::View<int*, TEST_EXECSPACE>;
+    view_type outer_view;
+
+    auto success = validate_existence(
+        [&]() {
+          view_type inner_view(
+              Kokkos::view_alloc(Kokkos::WithoutInitializing, "bla"), 8);
+          // Avoid testing the destructor
+          outer_view = inner_view;
+        },
+        [&](BeginFenceEvent event) {
+          return MatchDiagnostic{
+              event.descriptor().find(
+                  "fence after copying header from HostSpace") !=
+              std::string::npos};
+        });
+    ASSERT_TRUE(success);
+    listen_tool_events(Config::DisableAll());
+  }
+}
+
+TEST(TEST_CATEGORY, view_allocation_exec_space_int) {
+#ifdef KOKKOS_ENABLE_CUDA
+  if (std::is_same<typename TEST_EXECSPACE::memory_space,
+                   Kokkos::CudaUVMSpace>::value)
+    return;
+#endif
```

Comment:
Avoid doing this.  Unit test shows as running successfully even though it did nothing.

Reply:
Sure, we should use the tools we have. Fixed in 7ced72c85e3cbf6a4c0cd67abf5c822bc3bdd8ea